langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/internal/OpenAiClient.java

```diff
@@ -29,6 +30,8 @@ public abstract class OpenAiClient {
 
     public abstract SyncOrAsync<GenerateImagesResponse> imagesGeneration(GenerateImagesRequest request);
 
+    public abstract SyncOrAsync<ModelsListResponse> listModels();
```

Comment:
```suggestion
    public SyncOrAsync<ModelsListResponse> listModels() {
        throw new UnsupportedOperationException("not implemented yet");
    }
```
This is a breaking change, we need to provide a default implementation